composer.json

```diff
@@ -14,8 +14,8 @@
         "symfony/console": "^3.2",
         "symfony/yaml": "^3.2",
         "phpspec/php-diff": "^1",
-        "phpstan/phpstan": "^0.10",
-        "phpstan/phpstan-php-parser": "^0.10",
+        "phpstan/phpstan": "^0.9.0",
```

Comment:
```diff
-"phpstan/phpstan": "^0.9.0",
+"phpstan/phpstan": "~0.9.0",

Reply:
Good point. I'll fix `phpstan/phpstan-php-parser` and `phpstan/phpstan` to `~0.9.0`.

Let's revisit this when phpstan reaches `1.0.0`